server/seed.js

```diff
@@ -0,0 +1,112 @@
+const db = require('../database/index.js');
+
+
+/*********************************************************/
+/////////// Sample Properties /////////////////////////////
+/*********************************************************/
+
+const adjectives = ["Not Your Average ", "Super Soft ", "Sportsman's ", "Down to the Wire ", "City Sweat ", "Metal Vent ", "Basic ", "Fancy ", "Dryfit ", "Heavy ", "Extra Fun ", "Happy "];
+
+const itemTypes = ["Pants", "Shirt", "Tank-top", "Jacket", "Sweatshirt", "Shorts", "Yoga Mat", "Button Down Shirt", "Underwear", "Hat"];
+
+const descriptions = [
+  "Pellentesque adipiscing commodo elit at imperdiet dui accumsan sit amet.", "Id porta nibh venenatis cras sed.", "Imperdiet dui accumsan sit amet. Porttitor eget dolor morbi non arcu risus quis varius quam.", "Diam phasellus vestibulum lorem sed risus ultricies tristique nulla.", "Sociis natoque penatibus et magnis dis parturient montes.", "Nisl rhoncus mattis rhoncus urna neque viverra justo nec ultrices.", "In egestas erat imperdiet sed euismod nisi porta.", "Duis tristique sollicitudin nibh sit.", "Quam viverra orci sagittis eu volutpat odio facilisis mauris sit.", "Proin libero nunc consequat interdum varius sit amet mattis vulputate.", "Viverra justo nec ultrices dui sapien eget mi proin sed.", "Et egestas quis ipsum suspendisse ultrices.", "Augue ut lectus arcu bibendum at varius vel pharetra vel.", "Orci phasellus egestas tellus rutrum tellus.", "Ultrices eros in cursus turpis massa tincidunt dui.", "Egestas erat imperdiet sed euismod.", "Feugiat nisl pretium fusce id velit ut tortor pretium viverra.", "Aliquet lectus proin nibh nisl condimentum."
+];
+
+const fabrics = ["Wool", "Calico", "Silk", "Camel Hair", "Hemp", "Canvas", "Cashmere", "Linen", "Velvet"];
+
+const care = ["Machine wash cold", "Do not bleach", "Tumble dry low", "Do not iron", "Do not dry clean", "Imported", "Wash with like colors"];
+
+const designedFor = ["Office", "Travel", "Commute"];
+
+const fit = ["Relaxed fit", "Hip Length"];
+
+//sizes: array of nums
+const sizes = [0, 2, 4, 6, 8];
+
+//colors: array of colors
+const colors = ["White", "Black", "Blue", "Red", "Green"];
+
+/*******************************************************************************/
+/////////////////////// Generate Items ////////////////////////////////////////
+/*****************************************************************************/
+
+////////////// Helper Functions //////////////////
+
+const getRandomIndex = (length) => {
+  return Math.floor(Math.random() * length);
+};
+
+const getNRandomElements = (arr, n) => {
+  let _arr = [...arr];
+  return[...Array(n)].map( () => _arr.splice(Math.floor(Math.random() * _arr.length), 1)[0] );
```

Comment:
Your use of whitespace is very inconsistent. Please install a linter immediately so that your tooling can give you this feedback (instead of your PR reviewers).